<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVegetableScalePage</class>
 <widget class="QWidget" name="CVegetableScalePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="NLQT::CVegetNoiseValueWidget" name="scaleXYGroupBox">
     <property name="title">
      <string>Scale X/Y</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="NLQT::CVegetNoiseValueWidget" name="scaleZGroupBox">
     <property name="title">
      <string>Scale Z</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Bend Frequency Factor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="NLQT::CEditRangeFloatWidget" name="freqFactorWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>172</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLQT::CEditRangeFloatWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CVegetNoiseValueWidget</class>
   <extends>QGroupBox</extends>
   <header>vegetable_noise_value_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
